buildSrc/src/main/groovy/com/uber/okbuck/composer/android/AndroidLibraryRuleComposer.groovy

```diff
@@ -4,11 +4,14 @@ import com.uber.okbuck.core.model.android.AndroidLibTarget
 import com.uber.okbuck.core.model.android.AndroidTarget
 import com.uber.okbuck.core.model.base.RuleType
 import com.uber.okbuck.core.model.base.Target
+import com.uber.okbuck.core.util.KotlinUtil
 import com.uber.okbuck.core.util.RetrolambdaUtil
 import com.uber.okbuck.rule.android.AndroidLibraryRule
 
 final class AndroidLibraryRuleComposer extends AndroidBuckRuleComposer {
 
+     private static final String PLUGIN_OPTION = "plugin:org.jetbrains.kotlin.android:"
```

Comment:
Yeah definitely. Naming is hard, figured I'd come up with a better name before I'm ready to merge.